### DISCUSSION TITLE
PNP Filter Webpart not showing the Refiner

### DISCUSSION THREAD

**Discussion by LabibSalah** [microsoft-search/pnp-modern-search#3262] [General]
I had a managed property field which already successfully mapped in Admin Portal with the right crawled property. (see below the screenshot) but still not showing in the pnp filter Webpart.
Is there anyone in thje community who can help me ?
thanks a lot for your Support.
![image](https://github.com/microsoft-search/pnp-modern-search/assets/145577753/59354459-bd4d-43ba-ada4-9185d5871030)

**Comment by LabibSalah** [comment]
Einschränken means refinable

**Reply by kasperbolarsen** [reply]
Hi
Have you checked the results from the API using one of the tools here: https://microsoft-search.github.io/pnp-modern-search/using-query-tools-to-verify-issues/

The most common error is that the RefinableString01 is not selected or typed in the "Selected Poperties" field in the Results Web part. Can you confirm that RefinableString01 is indeed on that list?

**Comment by LabibSalah** [comment]
Hi Kasper,

thank you for your Reply.
In the pnp search result webpart i can see the RefinableString01 filed and I selected it, afterward i can select this property in the filter Webpart.(see below screenschots)
Despite the refinablestring correct crawled still not showing any values.
I did reindex for the document library many times.
![image](https://github.com/microsoft-search/pnp-modern-search/assets/145577753/f02e57b4-9380-468c-bece-d949b5f1a6ca)
![image](https://github.com/microsoft-search/pnp-modern-search/assets/145577753/74ae0035-dde1-4b6e-ac70-a08eecbbf783)

**Comment by kasperbolarsen** [comment]
1) As the refiners returned and displayed depends on the query you are using, I suggest searching for a specific item/document that you know should have a value in that property.
2) use the debug layout and/or one of the tools I linked to in the previous post in order to inspect the data returned from the API. Does RefinableString01 contains the values you expect?

**Comment by Funterrain** [comment]
Hi, in my case, the result web part only shows selected properties up to letter F. So it doesn't even get to the refine refinablestrings.
Any ideas why this is happening?

**Reply by kasperbolarsen** [reply]
Yes, the API that retrieve the managed properties is not reliable and there is nothing we can do about it. In those cases you will have to type in the managed properties yourself

**Reply by Funterrain** [reply]
You mean in that field? Do I just add the missing ones or can I somehow export what is already selected?

**Reply by kasperbolarsen** [reply]
Both actually.  You can export the current settings on the last page of the web part properties,  and you will need to type in eg. RefinableString 112 if that one is missing

**Reply by Funterrain** [reply]
Thanks, that helps a lot.
But I'm still struggling with the SiteTitle property. I want a Details List with a column "SiteTitle" (where in my SharePoint Hub the news is coming from). This is showing perfectly in PnP Search Results Webpart, but I am failing to use SiteTitle in PnP Search Filters ("No available filter to display"). I already tried with mapping, but no success.

<img width="1045" height="51" alt="image" src="https://github.com/user-attachments/assets/e3afd267-30ff-40e0-81fd-16f952efedf5" />

Any idea what is wrong?

**Reply by kasperbolarsen** [reply]
since the managed property Site Title is mapped to ows_SiteName :
<img width="1328" height="191" alt="image" src="https://github.com/user-attachments/assets/ddf4ca00-a940-457f-9795-94b8f30bb437" />

then you must map a RefinableString to that crawled property like here
<img width="1447" height="99" alt="image" src="https://github.com/user-attachments/assets/058e906b-d3b3-464a-a747-c4f94aed1fff" />


Then click on the site level reindexing request link, and then wait for the change to take effect ( up to 24 h sometimes). Once you see values in the RefinableString then you can add that RefinableString to the Selected Properties and config the Refiner web part to the use that RefinableString, remember that the Refiner wp must be mapped to the results wp AND the results wp to the Refiner wp.
